.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow release a new semantic version of npm package
+# For more information see: https://github.com/semantic-release/semantic-release/blob/2b94bb4e0967c705ab92deace342f9fecb02909d/docs/recipes/ci-configurations/github-actions.md
+
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+      - name: Use Node.js  🚧
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ secrets.NODE_VERSION }}
+          cache: ${{ !env.ACT && 'yarn' || '' }}
+      - name: Install Yarn 🚧
+        if: ${{ env.ACT }}
+        run: npm install -g yarn
+      - name: Build Package 🛠
+        run: yarn build
+      - name: Release 🚀
+        run: npx semantic-release --dry-run --debug
```

Comment:
left `--dry-run` and `-debug` in case we want to test it first :)

Reply:
probably we want to test it in the first place, right? As we discussed that we would like to understand who will appear as the author of a release on npm.

Reply:
NPM author would be me, since we're using my NPM token.
I'm curious about two things: 
1. who would be the author of the commits bumping the package.json versions.
2. what will be the contents of the published package.

Particularly, I'm worried about the second point, because not the entire codebase needs to be published, but only the `build` directory.

I think we can test it by running without `--dry-run` after setting release type to `pre-release` instead of `patch` in https://github.com/rangle/radius-tracker/blob/168a3c4ff48f4f83964cdb4b2a5e7491cd498118/release.config.js#L6

Reply:
Looks like the config is going to need `pkgRoot` option for the `npm` plugin:
https://github.com/semantic-release/npm#options